src/implementation/Client/GRPCClient/invoker.ts

```diff
@@ -54,7 +55,7 @@ export default class GRPCClientInvoker implements IClientInvoker {
     httpExtension.setVerb(HttpVerbUtil.convertHttpVerbStringToNumber(method));
 
     const msgSerialized = new Any();
-    msgSerialized.setValue(Buffer.from(JSON.stringify(data), "utf-8"));
+    msgSerialized.setValue(SerializerUtil.serializeGrpc(data));
```

Comment:
With this change, the content type in line 64 also needs to be set accordingly.

Reply:
One option is to skip setting the content type if we are directly sending bytes. @XavierGeerinck what do you think?

Reply:
Good Catch! Difficult question tbh... I am thinking of maybe introducing "MIME" type in the method signature and then pass that MIME type to the serializer util which can determine how to serialize based on it. Wdyt?

Reply:
Or do you think we should do the other way? i.e `SerializerUtil.serializeGrpc` returns serialized data along with a content type, based on where the data is JSON or bytes. Something like this (rough):
```ts
export function serializeGrpc(data: any): (Buffer, string) {
    let serialized = data;

    if (data instanceof Buffer) {
        serialized = data;
        contentType = "application/octet-stream"; # this looks like the right type but need to confirm
    } else {
        serialized = Buffer.from(JSON.stringify(data), "utf-8");
        contentType = "application/json";
    }

    return (serialized, contentType);
}
```

Reply:
But is a Buffer always octet stream?

Reply:
Need to do more research on that, let me get back on this tomorrow.

Reply:
From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#applicationoctet-stream),

> **application/octet-stream** 
This is the default for binary files. As it means unknown binary file, browsers usually don't execute it, or even ask if it should be executed.
...
Generic binary data (or binary data whose true type is unknown) is `application/octet-stream`.
...
Similarly, for binary documents without a specific or known subtype, `application/octet-stream` should be used. 

For Buffer type, indeed the data can be anything. It should be safe to use that IMO.

Reply:
Ok, let'se use it then!

Reply:
Hey @jilichao, can you please incorporate this into the PR? i.e. the `serializeGrpc` method can return both the data and the content-type based on if it's a Buffer or not.

Reply:
I just committed the changes. Could you please check if I understood correctly?

Reply:
@jilichao LGTM, just some minor comments.